src/Components/Grants/GrantsNew.js

```diff
@@ -20,7 +20,7 @@ export default function GrantsNew(props) {
     purpose: "",
     funding_org_id: null,
   });
-  const [fundingOrgs, setFundingOrgs] = useState([]);
+  const [fundingOrgs, setFundingOrgs] = useState([1, 2, 3]);
```

Comment:
Can this be removed?
```suggestion
  const [fundingOrgs, setFundingOrgs] = useState([]);
```